client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js

```diff
@@ -480,9 +486,11 @@ export const getFormErrors = createSelector(
 		const destinationCountryCode = form.destination.values.country;
 		const destinationCountryName = getCountryName( state, destinationCountryCode, siteId );
 		const shouldValidateOriginPhone = isCustomsFormRequired( state, orderId, siteId );
+		// If it's an international order, validate the destination phone as well.
```

Comment:
One note here for `isCustomsFormRequired()` function. There is a special case that custom is required for military addresses. In this case, phone number will now also be required. I guess that's okay?

Reply:
I think the upside of re-using the function outweighs the downside. We could keep an eye out for complaints about those specific addresses(it's a tiny tiny percentage of shipments).